encord/client.py

```diff
@@ -1065,5 +1074,82 @@ def __set_project_ontology(self, ontology: Ontology) -> bool:
         payload = {"editor": ontology.to_dict()}
         return self._querier.basic_setter(Project, uid=None, payload=payload)
 
+    def get_project_labels_consensus(
+        self,
+        comparing_projects: List[EncordClientProject],
+        ontology_feature_node_hash_list: List[str],
+        threshold: float = 0.7,
+    ) -> Dict[str, LabelAnnotationMetrics]:
+        """
+        Calculate performance metrics of a label generating agent when compared with consensus reached by other agents.
+
+        Args:
+            self:
+                The project to be evaluated. Also known as baseline project.
+            comparing_projects:
+                The list of projects where consensus is extracted.
+            ontology_feature_node_hash_list:
+                The list of feature node hashes denoting object classes from baseline project ontology to be evaluated.
+            threshold:
+                Used in objects comparison. Two objects' instances are considered the same object if their Jaccard
+                similarity coefficient is greater or equal than the threshold; otherwise, they represent distinct objects.
+
+        Returns:
+            Label annotation metrics of object classes from baseline project indexed by their feature node hashes.
```

Comment:
Can we rewrite this to something like "A dict of ontology feature node hashes to `LabelAnnotationMetrics`.